### PR TITLE
docs: fix typos

### PR DESCRIPTION
**Pull request by romashka-btc**
# Fix: Corrected typos in source and documentation files

## Changes
1. **Corrected typo in `mempool/cat/spec.md:78`**:
   - "reponse" corrected to "response".

2. **Corrected typo in `mempool/cat/spec.md:39`**:
   - "recieved" corrected to "received".

3. **Corrected typo in `rpc/client/mock/client.go:57`**:
   - "apporiate" corrected to "appropriate".

## Purpose
- Improved code and documentation accuracy by fixing typos.

### CODE REVIEW

**Review by rootulp — APPROVED** [review]

**Comment by romashka-btc** [comment]
@rootulp , Hi! Could you please take a look at my pull request? Your feedback is very important for moving forward. Thank you so much!

**Comment by rootulp** [comment]
> @rootulp , Hi! Could you please take a look at my pull request? Your feedback is very important for moving forward. Thank you so much!

I already reviewed and approved it.This PR needs one more approval before it can be merged.

**Comment by romashka-btc** [comment]
> > @rootulp , Hi! Could you please take a look at my pull request? Your feedback is very important for moving forward. Thank you so much!
> 
> I already reviewed and approved it.This PR needs one more approval before it can be merged.

ok, thank you !

**Review by rach-id — APPROVED** [review]

**Pull request merged**
